ten-simple-rules-dockerfiles.Rmd

```diff
@@ -32,15 +33,28 @@ author:
     email: tony.hirst@open.ac.uk
     affiliation: "Department of Computing and Communications, The Open University, GB"
     orcid: 0000-0001-6921-702X
+  - name: Benjamin Evans
+    # https://github.com/bdevans
+    email: benjamin.evans@bristol.ac.uk
+    affiliation: "School of Psychological Science, University of Bristol, GB"
```

Comment:
For consistency, it should be "School of Psychological Science, University of Bristol, Bristol, GB".

Reply:
Also:
1.  "Stanford University, Stanford, CA, US" should be "Stanford University, Stanford, CA, USA".
2. "University of Washington, Seattle, USA" should be "University of Washington, Seattle, WA, USA" if you include the state or else remove CA from above.

Reply:
My affiliation is otherwise correct, thank you.

Reply:
Thanks, well spotted!